### PR TITLE
Bug 1738360: asset/installconfig: use new link to OCM pull-secret page

### PR DESCRIPTION
**Pull request by gregsheremeta**

### CODE REVIEW

File: pkg/asset/installconfig/pullsecret.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ func (a *pullSecret) Generate(asset.Parents) error {
 		{
 			Prompt: &survey.Password{
 				Message: "Pull Secret",
-				Help:    "The container registry pull secret for this cluster, as a single line of JSON (e.g. {\"auths\": {...}}).\n\nYou can get this secret from https://cloud.openshift.com/clusters/install#pull-secret",
+				Help:    "The container registry pull secret for this cluster, as a single line of JSON (e.g. {\"auths\": {...}}).\n\nYou can get this secret from https://cloud.openshift.com/openshift/install/pull-secret",
```
Comment (wking) [inline pkg/asset/installconfig/pullsecret.go]: Should be cloud.redhat.com now, no?

Comment (gregsheremeta) [reply pkg/asset/installconfig/pullsecret.go]: ugh, of course. Good eye Trevor, thank you. me~~
fixed in ae44ae1

**Comment by wking** [comment]
```
error: unable to upload the new image manifest: unknown blob
```

CI flake.

/retest

**Pull request merged**